frontend/src/components/Image/styled.tsx

```diff
@@ -1,29 +1,14 @@
 import styled from 'styled-components';
 
-import { SIZE } from 'constants/sizes';
-
-interface Size {
-  size?: string;
-}
-
-export const ImageContainer = styled('div')<Size>(({ size }) => ({
```

Comment:
Why did you remove size prop for image? If we need to use image of other size anywhere else in future we won't be able to use it

Reply:
For this we have wrapper. I suppose that each image should have wrapper. And this wrapper is settling width, height and otherwise props. Image have css props as like `object-fit`  for work with this.